scrivito_codemirror_editor.gemspec

```diff
@@ -2,7 +2,7 @@ Gem::Specification.new do |s|
   s.author = 'Scrivito'
   s.description = 'Make CodeMirror available as a Scrivito editor'
   s.email = 'info@scrivito.com'
-  s.files = Dir['READ*', 'LIC*', 'app/**/*', 'lib/**/*']
+  s.files = Dir['READ*', 'LIC*', 'app/**/*', 'lib/**/*', 'vendor/**/*']
```

Comment:
* [ ] The CodeMirror code looks like the addons rely on the main script to be loaded first. If I read this correctly, we should make the order explicit.

Reply:
This is how `codemirror-rails` defines its "files":

https://github.com/fixlr/codemirror-rails/blob/cb5e9eee1bac8b8b6933d330591a5f83100094d8/codemirror-rails.gemspec#L13

So I think we're ok with the current approach.

Reply:
I see. Ruby even turned to sorted at some point: https://bugs.ruby-lang.org/issues/8709